lib/modules/manager/bazel/update.spec.ts

```diff
@@ -1,7 +1,7 @@
 import crypto from 'crypto';
 import * as httpMock from '../../../../test/http-mock';
 import type { UpdateType } from '../../../config/types';
-import { updateDependency } from '.';
+import { updateDependency } from './update';
```

Comment:
```suggestion
import { updateDependency } from '.';
```
always import from index (branch coverage), or isn't it no longer public